cfg/Instructions.h

```diff
@@ -93,22 +129,22 @@ class Send final : public Instruction {
     bool isPrivateOk;
     std::shared_ptr<core::SendAndBlockLink> link;
 
-    Send(core::LocalVariable recv, core::NameRef fun, core::LocOffsets receiverLoc,
-         const InlinedVector<core::LocalVariable, 2> &args, InlinedVector<core::LocOffsets, 2> argLocs,
-         bool isPrivateOk = false, const std::shared_ptr<core::SendAndBlockLink> &link = nullptr);
+    Send(LocalRef recv, core::NameRef fun, core::LocOffsets receiverLoc, const InlinedVector<LocalRef, 2> &args,
+         InlinedVector<core::LocOffsets, 2> argLocs, bool isPrivateOk = false,
+         const std::shared_ptr<core::SendAndBlockLink> &link = nullptr);
 
-    virtual std::string toString(const core::GlobalState &gs) const;
-    virtual std::string showRaw(const core::GlobalState &gs, int tabs = 0) const;
+    virtual std::string toString(const core::GlobalState &gs, const CFG &cfg) const;
+    virtual std::string showRaw(const core::GlobalState &gs, const CFG &cfg, int tabs = 0) const;
 };
 CheckSize(Send, 160, 8);
```

Comment:
Why didn't size change here?

Reply:
Not sure; I'd think that `VariableUseSite` would be smaller. Will investigate.

Reply:
`VariableUseSite`'s size did not change. It contains a 16 byte `TypePtr` and a 4 byte `LocalRef` so, with padding, it's still 24 bytes.